server/api/observation/type/observation-type.model.ts

```diff
@@ -0,0 +1,9 @@
+import { BaseEntity, Entity, PrimaryColumn, Unique } from 'typeorm';
+
+@Entity()
+@Unique(['type'])
+export class ObservationType extends BaseEntity {
+
+  @PrimaryColumn({ length: 100 })
```

Comment:
This is declaring observation string as the primary key? Is this the string for the observation (like a description) itself or intended as an actual primary key?

Reply:
ObservationType was removed. The latest WIP code is here: https://github.com/mystatus/covid19-reports/tree/add-observation-type-fk/server/api/observation